Dockerfile

```diff
@@ -31,20 +31,18 @@ RUN set -x; yum -y update && \
 		gcc gcc-c++ make bison flex file systemd-devel zlib-devel gtest-devel rpm-build java-11-openjdk-devel \
 		expect rpm-sign zip wget && \
 		yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
-		yum install -y cmake3 && \
-		ln -fs cmake3 /usr/bin/cmake && \
-		# Download and install CMake manually (zypper only gives us CMake 3.5, but we need >= 3.12)
-		wget https://github.com/Kitware/CMake/releases/download/v3.25.2/cmake-3.25.2-linux-x86_64.sh -O cmake.sh && \
-		(echo '4d98de8d605da676e71a889dd94f80c76abb377fade2f21e3510e62ece1e1ada  cmake.sh' | sha256sum --check) && \
-		bash cmake.sh --skip-license --prefix=/usr/local
+		wget https://github.com/Kitware/CMake/releases/download/v3.25.2/cmake-3.25.2-linux-$(uname -m).sh -O cmake.sh && \
+	((echo '4d98de8d605da676e71a889dd94f80c76abb377fade2f21e3510e62ece1e1ada  cmake.sh' | sha256sum --check) || \
+	(echo '73a35cab2174a3eb8f35083d55c80871185dc3808f3dae3558cd5fbdb29a4614  cmake.sh' | sha256sum --check)) && \
```

Comment:
Is this because the ARM package has a different checksum? Would it be possible to make that explicit (e.g., define a map from architecture to checksum and key it by the parameter) instead?

Reply:
Yeah, ARM has a different checksum. I'll try making it a map.

Reply:
Changed to a map.